Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -296,4 +302,20 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
       destination.appCenter = appCenter
     }
   }
+  
+  func showToast(message : String) {
```

Comment:
When I run this on macos, the toast may be hidden because of scrolling position. I guess that's more of an Android, term, it's better to use `UIAlertController`